build-tools/installers/create-installers.targets

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\Configuration.props" />
+  <Import Project="..\scripts\XAVersionInfo.targets" />
+  <UsingTask AssemblyFile="..\..\bin\Build$(Configuration)\xa-prep-tasks.dll"   TaskName="Xamarin.Android.BuildTools.PrepTasks.ReplaceFileContents" />
+  <PropertyGroup>
+    <RootBuildDir>$(XamarinAndroidSourcePath)\bin\$(Configuration)\</RootBuildDir>
+    <FrameworkSrcDir>$(RootBuildDir)\lib\xamarin.android\xbuild-frameworks\MonoAndroid</FrameworkSrcDir>
+    <MSBuildSrcDir>$(RootBuildDir)\lib\xamarin.android\xbuild\Xamarin\Android</MSBuildSrcDir>
+    <FirstFrameworkVersion>v4.4</FirstFrameworkVersion>
+    <BclFrameworkVersion>v1.0</BclFrameworkVersion>
+    <LibExtension Condition=" '$(HostOS)' == 'Darwin' ">dylib</LibExtension>
+    <LibExtension Condition=" '$(HostOS)' == 'Linux' ">so</LibExtension>
+    <LibExtension Condition=" '$(HostOS)' == 'Windows' ">dll</LibExtension>
+  </PropertyGroup>
+  <ItemGroup>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\*\AndroidApiInfo.xml"></FrameworkFiles>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\*\Mono.Android.*"></FrameworkFiles>
+    <FrameworkFiles Include="$(FrameworkSrcDir)\**\FrameworkList.xml"></FrameworkFiles>
+    <FirstFrameworkFiles Include="$(FrameworkSrcDir)\$(FirstFrameworkVersion)\Mono.Android.Export.dll"></FirstFrameworkFiles>
+    <FirstFrameworkFiles Include="$(FrameworkSrcDir)\$(FirstFrameworkVersion)\OpenTK*.dll"></FirstFrameworkFiles>
+    <BCLFrameworkFiles Include="$(FrameworkSrcDir)\$(BclFrameworkVersion)\**\*" Exclude="**\*.pdb;**\libZipSharp.dll.config"></BCLFrameworkFiles> 
```

Comment:
If we want to explicitly specify files, which we should, we can use `src/mono-runtimes/ProfileAssemblies.projitems` to provide the list of assemblies to install:

```xml
<Import Project="..\..\src\mono-runtimes\ProfileAssemblies.projitems" />
<ItemGroup>
  <BCLFacadeFiles Include="@(MonoFacadeAssembly->'$(FrameworkSrcDir)\$(BclFrameworkVersion)\Facades\%(Identity)')" />
  <BCLFrameworkFiles Include="@(MonoProfileAssembly->'$(FrameworkSrcDir)\$(BclFrameworkVersion)\%(Identity)')" />
</ItemGroup>
```

Reply:
@jonpryor I've made this change, however there are a few extra files in `*MonoAndroid\v1.0\` in our current .pkg which are not listed in `src\mono-runtimes\ProfileAssemblies.projitems`.

See https://github.com/xamarin/xamarin-android/pull/2733/files#diff-eeee5134063ea9bc7a4c69343d8f5208R26 lines 26-30 for those items.

Should some (all) of these be moved into `src\mono-runtimes\ProfileAssemblies.projitems`?

Reply:
> Should some (all) of these be moved into `src\mono-runtimes\ProfileAssemblies.projitems`?

No.  `@(MonoFacadeAssembly)`/`@(MonoProfileAssembly)` are files that come *from **mono***, and are part of the mono bundle, and are used to help determine whether or not the mono bundle is *valid*, e.g. if there's a file listed in one of those item groups but *isn't* in the mono bundle, then the bundle is invalid and we should rebuild mono.

The other assemblies -- `Java.Interop.dll`, `Mono.Data.Sqlite.dll.config`, `Mono.Posix.dll`, `System.EnterpriseServices.dll`, `Xamarin.Android.NUnitLite.dll` -- are built *by the **xamarin-android*** repo, *not* mono.  (`Mono.Posix.dll` should *eventually* be built by mono, but that hasn't happened yet, and a case could be made for `System.EnterpriseServices.dll`.  The other files are unlikely to ever be part of mono.)

These files will *not* be included in the mono bundle.

As such, they should *not* be included within the `@(MonoFacadeAssembly)`/`@(MonoProfileAssembly)` item groups.

Reply:
Makes sense to me, I'll keep the extra explicit item additions as is.